### ISSUE TITLE
`no-unregistered-classes` reports Tailwind default classes as unregistered

### ISSUE THREAD

**Issue by lukasvice** [schoero/eslint-plugin-better-tailwindcss#201]
Hi,

I’m trying out this plugin and I’m not sure if the following behavior is intended:

When using the `no-unregistered-classes` rule, Tailwind’s default utility classes are reported as unregistered. For example:

```html
<div class="gap-4"></div>
```

This triggers an error:

`Unregistered class detected: gap-4`

It’s true that `gap-4` is not defined in my `theme.css`, but it is a default class provided by Tailwind.

I can resolve the issue by manually adding

```css
--spacing-4: 0.25rem;
```

to the theme.

**Expected behavior**

Default Tailwind classes (e.g., `gap-4`) should be accepted automatically, unless the corresponding CSS variables are explicitly unset (e.g., `--spacing-*: initial`).

**Comment by lukasvice** [comment]
Thank you for confirming that this is not the intended behavior. I think I have set the entry point correctly. I am currently updating and restructuring my ESLint configuration, so I will look into this further and let you know what I find out.
